charts/camunda-platform/templates/connectors/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.connectors.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "connectors.fullname" . }}
+  labels: {{- include "connectors.labels" . | nindent 4 }}
+  annotations: {{- toYaml .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "connectors.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "connectors.labels" . | nindent 8 }}
+        {{- if .Values.connectors.podLabels }}
+        {{- toYaml .Values.connectors.podLabels | nindent 8 }}
+        {{- end }}
+        {{- if .Values.connectors.podAnnotations }}
+        annotations: {{- toYaml  .Values.connectors.podAnnotations | nindent 8 }}
+        {{- end }}
+    spec:
+{{/*      imagePullSecrets:*/}}
+{{/*        {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}*/}}
+      containers:
+        - name: connectors
+          image: {{ include "camundaPlatform.imageByParams" (dict "base" .Values.global "overlay" .Values.connectors) }}
+          imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+          {{- if .Values.connectors.containerSecurityContext }}
+          securityContext: {{- toYaml .Values.connectors.containerSecurityContext | nindent 10 }}
+          {{- end }}
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+          env:
+          {{- if .Values.connectors.env}}
+            {{ .Values.connectors.env | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if eq .Values.connectors.inbound.mode "disabled" }}
+            - name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
+              value: "{{ include "zeebe.names.gateway" . | replace "\"" "" }}:{{ index .Values "zeebe-gateway" "service" "gatewayPort" }}"
+            - name: ZEEBE_CLIENT_SECURITY_PLAINTEXT
+              value: "true"
+            - name: CAMUNDA_CONNECTOR_POLLING_ENABLED
+              value: "false"
+            - name: CAMUNDA_CONNECTOR_WEBHOOK_ENABLED
+              value: "false"
+            - name: SPRING_MAIN_WEB-APPLICATION-TYPE
+              value: "NONE"
+          {{- end }}
+          {{- if eq .Values.connectors.inbound.mode "credentials" }} # Not implemented yet
+            - name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
```

Comment:
Where this env var `ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS` is used?
Is that Zeebe config or Connectors config?

Reply:
That's Connector's environment variable. The intent here is to replicate [this behavior](https://github.com/camunda/camunda-platform/blob/main/docker-compose-core.yaml#L70).